pkg/driver/client.go

```diff
@@ -861,23 +861,23 @@ func (conn *BackendConnection) Ping(ctx context.Context) (err error) {
 //
 // 1. if the server closes the connection when no command is in flight:
 //
-//   1.1 unix: WriteComQuery will fail with a 'broken pipe', and we'll
-//       return CRServerGone(2006).
+//		1.1 unix: WriteComQuery will fail with a 'broken pipe', and we'll
+//		    return CRServerGone(2006).
 //
-//   1.2 tcp: WriteComQuery will most likely work, but ReadComQueryResponse
-//       will fail, and we'll return CRServerLost(2013).
+//		1.2 tcp: WriteComQuery will most likely work, but ReadComQueryResponse
+//		    will fail, and we'll return CRServerLost(2013).
 //
-//       This is because closing a TCP socket on the server side sends
-//       a FIN to the client (telling the client the server is done
-//       writing), but on most platforms doesn't send a RST.  So the
-//       client has no idea it can't write. So it succeeds writing Content, which
-//       *then* triggers the server to send a RST back, received a bit
-//       later. By then, the client has already started waiting for
-//       the response, and will just return a CRServerLost(2013).
-//       So CRServerGone(2006) will almost never be seen with TCP.
+//		    This is because closing a TCP socket on the server side sends
```

Comment:
Too much space here, is there a purpose for that?

Reply:
code format